build/uno.ui.targets

```diff
@@ -8,29 +8,23 @@
 
 	<Import Project="$(SourceGeneratorBasePath)Uno.UI.SourceGenerators.props" />
 	<Import Project="$(UnoUIMSBuildTasksTargetPath)Uno.UI.Tasks.targets" />
-	<Target Name="_UnoFeatureDefines"
```

Comment:
This needs to stay as a target. Using a global property group makes the defined constants appear in builds constants VS UI field of an end user project using Uno, and ultimately those properties end up appearing that project's own `DefineConstants` property.

We can better document the reason for this target, though.

Reply:
Done.